backend/hitas/tests/apis/test_api_apartment.py

```diff
@@ -496,6 +499,10 @@ def _test_max_prices(
         mpi_factory.create(month=now, value=250)
         SurfaceAreaPriceCeilingFactory.create(month=now, value=3000)
 
+    if old_hitas_ruleset:
+        ap.housing_company.financing_method.old_hitas_ruleset = old_hitas_ruleset
```

Comment:
This can be defined when creating the apartment like so:
```
ApartmentFactory.create(
    ...,
    building__real_estate__housing_company__financing_method__old_hitas_ruleset = old_hitas_ruleset
)
```

Reply:
Done